scripts/find_untracked_release_tests.py

```diff
@@ -0,0 +1,88 @@
+import os
+import yaml
+import tempfile
+import re
+import pprint
+
+pp = pprint.PrettyPrinter(indent=4)
+tempdir = tempfile.mkdtemp()
+
+def ci_covered_release_tests(root_dir, pre_fix="repos/"):
+    covered_release_test_names = set()
+    for path, _, files in os.walk(root_dir):
+        for filename in files:
+            filepath = os.path.join(path, filename)
+            with open(filepath, "r") as f:
+                content = yaml.safe_load(f)
+
+            for item in content:
+                if "path" in item.keys():
+                    test_path = item["path"]
+                    
+                    # Remove pre-fix
+                    if test_path.startswith(pre_fix):
+                        test_path = test_path[len(pre_fix):]
+                        covered_release_test_names.add(test_path)
+
+    return covered_release_test_names
+
+def find_existing_examples(repo_name, github_addr, examples_dir, require_entrance=True):
+    repo_dir = f"{tempdir}/{repo_name}/"
+    os.system(f"git clone {github_addr} {repo_dir}")
+    
+    existing_examples = set()
+    examples_root_dir = os.path.join(repo_dir, examples_dir)
+    for path, _, files in os.walk(examples_root_dir):
+        for filename in files:
+            if not filename.endswith(".py"):
+                continue
+
+            filepath = os.path.join(path, filename)
+            if require_entrance:
+                with open(filepath, "r") as f:
+                    fcontent = f.read()
+                    pattern = r"if __name__\s*==\s*[\\\'\\\"]__main__[\\\'\\\"]\s*:"
```

Comment:
Not all examples have this structure. Although this is guarded by a condition variable but I can see nowhere it is specified a `False` value. Is this intended?

Reply:
It's almost impossible to distinguish between "util files" and files which actually executes the demo due to the nature of python code. Let me make `require_entrance` an option, so people can decide whether they want to manually pick out the util files.